KsApi/models/graphql/adapters/Backing+BackingFragmentTests.swift

```diff
@@ -411,6 +418,10 @@ private func backingDictionary() -> [String: Any] {
       "deadlineAt": 1620478771,
       "description": "Dark Fantasy Novel & Tarot Cards",
       "finalCollectionDate": null,
+      "friends": {
+        "__typename": "ProjectBackerFriendsConnection",
+        "edges": []
```

Comment:
I wonder if it would be a more useful test if we had friends returning in this array?  I know it's a tough situation to mock while using the app on staging though.

Reply:
Yea, fair point, I left this out only because `Backing+BackingFragmentTests` is only really testing the `Backing` object creation, `friends` is on `project.personalization` which is out of testing scope here. Also trying not to bloat up this `jsonDictionary`.